System/GraphicalPrimitive.cpp

```diff
@@ -431,18 +531,26 @@ namespace RTE {
 
 		if (!g_SceneMan.SceneWrapsX() && !g_SceneMan.SceneWrapsY()) {
 			Vector drawStart = m_StartPos - targetPos;
-			pivot_scaled_sprite(drawScreen, bitmapToDraw, drawStart.GetFloorIntX(), drawStart.GetFloorIntY(), bitmapToDraw->w / 2, bitmapToDraw->h / 2, ftofix(rotation.GetAllegroAngle()), ftofix(1.0));
+			if (m_BlendMode > DrawBlendMode::NoBlend) {
+				pivot_scaled_sprite_trans(drawScreen, bitmapToDraw, drawStart.GetFloorIntX(), drawStart.GetFloorIntY(), bitmapToDraw->w / 2, bitmapToDraw->h / 2, ftofix(rotation.GetAllegroAngle()), ftofix(1.0));
```

Comment:
What is the ftofix(1.0) for here?

Reply:
Hover the function and find out. 
![judge-right](https://user-images.githubusercontent.com/50018358/209763257-0e2d6ad9-4f8f-4c4d-bf58-ef06ad7d85f7.png)

Reply:
lol u basterd

Fair enough, it's scale, that makes sense